chai/src/main/java/com/droidconke/chai/typography/README.md

```diff
@@ -0,0 +1,45 @@
+# Typography
+This is a simple doc that should be used in line with the typography tokens in the design system

```

Comment:
Replace with: 
These are the typography tokens in the design system.

Reply:
Done...